README.md

```diff
@@ -56,65 +56,67 @@ The following table lists the configurable parameters of the Codacy chart and th
-| `global.githubEnterprise.enabled`       | Enable githubEnterprise                                                                                      | `nil`           |
-| `global.githubEnterprise.hostname`      | Hostname of githubEnterprise instance                                                                        | `nil`           |
-| `global.githubEnterprise.protocol`      | Protocol of githubEnterprise instance                                                                        | `nil`           |
-| `global.githubEnterprise.port`          | Port of githubEnterprise instance                                                                            | `nil`           |
-| `global.githubEnterprise.isPrivateMode` | Status of private mode on githubEnterprise instance                                                          | `nil`           |
-| `global.githubEnterprise.disableSSL`    | Disable certificate validation on interaction with githubEnterprise instance                                 | `nil`           |
-| `global.gitlabEnterprise.enabled`       | Enable gitlabEnterprise                                                                                      | `nil`           |
-| `global.gitlabEnterprise.hostname`      | Hostname of gitlabEnterprise instance                                                                        | `nil`           |
-| `global.gitlabEnterprise.protocol`      | Protocol of gitlabEnterprise instance                                                                        | `nil`           |
-| `global.gitlabEnterprise.port`          | Port of gitlabEnterprise instance                                                                            | `nil`           |
-| `global.bitbucketEnterprise.enabled`    | Enable bitbucketEnterprise                                                                                   | `nil`           |
-| `global.bitbucketEnterprise.hostname`   | Hostname of bitbucketEnterprise instance                                                                     | `nil`           |
-| `global.bitbucketEnterprise.protocol`   | Protocol of bitbucketEnterprise instance                                                                     | `nil`           |
-| `global.bitbucketEnterprise.port`       | Port of bitbucketEnterprise instance                                                                         | `nil`           |
-| `global.features.cloneSubmodules`       | Enable sharing of configuration files for the analysis tools placed on git submodules                        |                 |
-| `false`                                 |                                                                                                              |                 |
+| Parameter                                             | Description                                                                                                  | Default         |
```

Comment:
@bmbferreira why did this change?

Reply:
Use the remark-lint as formater on vscode

Reply:
codacy complained about it